best-practices/ml-platform/examples/cluster/shared_config/workloads_variables.tf

```diff
@@ -0,0 +1,11 @@
+locals {
+  manifests_directory = abspath("${path.module}/../manifests")
+}
+
+variable "enable_kueue" {
+  default = true
+}
+
+variable "kueue_version" {
+  default = "0.8.0"
```

Comment:
The bash scripts are using v0.8.1. It would be interesting to line up the different versions